bin/dispatch-conf

```diff
@@ -155,11 +155,25 @@ class dispatch:
             if not os.path.isdir(os.path.dirname(path)):
                 continue
 
+			# check whether find command is the GNU version
+			# if not, use BSD-style find arguments
+            s = subprocess.run(["find", "--version"], capture_output=True)
+            if s.returncode == 1:
+                gnu_find = False
+            else:
+                gnu_find = True
+
             basename = "*"
-            find_opts = ["-name", ".*", "-type", "d", "-prune", "-o"]
+            if gnu_find == True:
+                find_opts = ["-name", ".*", "-type", "d", "-prune", "-o"]
+            else:
+                find_opts = ["--", "-name", ".*", "-type", "d", "-prune", "-o"]
```

Comment:
Requiring `--` before option names sounds like a bug, especially that it makes the invocation incompatible with POSIX.